packages/swap/src/event-handlers/networkDepositFinalised.ts

```diff
@@ -86,8 +86,8 @@ export const networkDepositFinalised = async ({ prisma, event, block }: EventHan
       where: { nativeId: swapRequestId },
       data: {
         depositAmount: amount.toString(),
-        depositReceivedAt: new Date(block.timestamp),
-        depositReceivedBlockIndex: `${block.height}-${event.indexInBlock}`,
+        depositFinalisedAt: new Date(block.timestamp),
+        depositFinalisedBlockIndex: `${block.height}-${event.indexInBlock}`,
```

Comment:
Ok it's actually a bit trickier because we have to think about boosted liquidity deposits. We have to make sure we have a matching swap request for the prewitness deposit id

Reply:
that should already be handled below i think - were updating the `SwapRequest` that matches the `prewitnessedDepositId`